main/realmain.cc

```diff
@@ -804,25 +730,27 @@ int realmain(int argc, char *argv[]) {
             gs->suppressErrorClass(core::errors::Resolver::StubConstant.code);
             gs->suppressErrorClass(core::errors::Resolver::RecursiveTypeAlias.code);
 
-            indexed = pipeline::package(*gs, move(indexed), opts, *workers);
+            // Remove all package files
+            auto it = std::remove_if(indexed.begin(), indexed.end(),
+                                     [&gs](auto &file) -> bool { return file.file.data(*gs).isPackage(); });
+            indexed.erase(it, indexed.end());
```

Comment:
from slack:

"should we have solved this with `--ignore=__package.rb`?"

"ah true, we can do that"